### ISSUE TITLE
Importing "index.js" in browser does not work

### ISSUE THREAD

**Issue by jehon** [BrodaNoel/duix#16]
By importing

```
import deepEqual from './utils/deepEqual';
```

The import does not work in browser, because the extension is not set.

Use instead:
```
import deepEqual from './utils/deepEqual.js';
```

**Comment by jehon** [comment]
Could you release a version to fix this one ? 2.0.2 should be good enough for me
Thanks

**Comment by BrodaNoel** [comment]
@jehon done!
